channels/alpha

```diff
@@ -7,6 +7,8 @@ spec:
     - name: kope.io/k8s-1.5-debian-jessie-amd64-hvm-ebs-2017-01-09
       providerID: aws
       kubernetesVersion: ">=1.5.0"
+    - providerID: gce
+      name: "cos-cloud/cos-stable-56-9000-84-2"
```

Comment:
For some reason, now I get the COS images with just a `gcloud compute images list`. Not sure if it's because they're now available because I "activated them" by specifying the project cos-cloud, or because I updated my gcloud. I bet once I pointed my gcloud utils at that project, they became available...